src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/commands/mobile/get_interior_vehicle_data_request.cc

```diff
@@ -128,14 +128,18 @@ void GetInteriorVehicleDataRequest::ProcessResponseToMobileFromCache(
       auto extension = RCHelpers::GetRCExtension(*app);
       DCHECK(extension);
       extension->SubscribeToInteriorVehicleData(module);
+      app->UpdateHash();
```

Comment:
@IGapchuk Do we need it here? I can see UpdateHash() here https://github.com/LuxoftSDL/sdl_core/pull/87/files#diff-64546aa83f6f7211621135e409367272R48 in the extension->SubscribeToInteriorVehicleData()